code/modules/antagonists/changeling/changeling.dm

```diff
@@ -164,8 +163,8 @@
 		return
 
 	geneticpoints -= thepower.dna_cost
-	purchasedpowers += thepower
-	thepower.on_purchase(owner.current)
+	purchasedpowers += thepower//maybe New() the new button here?
```

Comment:
stop with the dumb comments

Reply:
my b a lot of these were for self reference to get a grasp on existing ling code and some were TODOs that i forgot to put a TODO on